packages/hop-node/src/watchers/classes/BaseWatcher.ts

```diff
@@ -36,7 +36,7 @@ class BaseWatcher extends EventEmitter implements IBaseWatcher {
   order: () => number = () => 0
   started: boolean = false
   pollIntervalSec: number = 10 * 1000
-  resyncIntervalSec: number = 10 * 60 * 1000
+  resyncIntervalSec: number = 60 * 1000
```

Comment:
I think both `pollIntervalSec` and `resyncIntervalSec` should be renamed to use `Ms` instead of  `Sec` here since it's being multiplied by `1000`

Reply:
Fix 1b4fc481ea7398f99fbd7364d4ae3a6a38eaf4dc